src/AppController.m

```diff
@@ -186,7 +186,6 @@ -(void)awakeFromNib
 	mainWindow.title = self.appName;
 	[NSApplication sharedApplication].delegate = self;
 	mainWindow.minSize = NSMakeSize(MA_Default_Main_Window_Min_Width, MA_Default_Main_Window_Min_Height);
-    [mainWindow setAllowsConcurrentViewDrawing:YES];
```

Comment:
This won't disable concurrent drawing as the default value is `YES`. https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSWindow_Class/index.html#//apple_ref/occ/instp/NSWindow/allowsConcurrentViewDrawing